brewerdigital-llc/VivoCare
# fixed image update bug.

# Description
This PR fixes a query invalidation bug preventing the header from updating the photo when necessary.

# Closes issue(s)
photo update bug

# Steps to Test
1. Run app via any method
2. Update photo
3. See photo update on header

# Changes include

# Checklist
